scss/civicrm/common/_common.scss

```diff
@@ -19,4 +19,5 @@
 @import 'help';
 @import 'pagination';
 @import 'notifications';
+@import 'wizard';
```

Comment:
you are not creating this partial, so how come it wasn't in `common.scss` already? where was it used?

Reply:
The wizard itself was styled by Mukesh but he did not include it in this file. As I needed to work with the wizard, we agreed with Mukesh that I shall import this file and commit it.

Reply:
This wizard was styled as part of CiviHR and was forgotten to be added in this file when merging. We are just adding it here again. @mihaelmladenov your latest pull to develop should have added this already though.